tools/paths.py

```diff
@@ -24,7 +24,9 @@
 BUILD_DIR = getenv("MBED_BUILD_DIR") or BUILD_DIR
 
 # Embedded Libraries Sources
-LIB_DIR = join(ROOT, "features/unsupported")
+FEATURES_DIR = join(ROOT, "features")  
```

Comment:
Looks like mbed 2 specific testing was removed so this should be "features/unsupported" as it was